packages/ld-key-pair/KeyPair.ts

```diff
@@ -0,0 +1,103 @@
+/**
```

Comment:
I think we could probably split out these interfaces into their own interface files, and to have a keypair folder, to make a little bit more organized.  Also not sure if we wanted to go further with some of these any types to build out the interfaces more.